### DISCUSSION TITLE
스터디 2기 사이드 프로젝트

### DISCUSSION THREAD

**Discussion by DaleSeo** [DaleStudy/leetcode-study#492] [공지사항]
지난 정기 모임에서 자세히 안내드렸던 것처럼 스터디 1기와 마찬가지로 2기에서도 지원자를 받아서 사이드로 협업 프로젝트를 하나 진행하려고 합니다. 15주 내내 주구장창 코딩 테스트 문제만 풀다보면 좀 지치는 감도 있는데, 중간에 사이드 프로젝트를 해주면 리프레쉬가 되고 동기 부여도 되서 괜찮더라고요.

참고로 1기 때는 저 포함 7분 정도 모여서 약 8주 동안 Web Components라는 비교적 새로운 웹 기술을 사용하여 [달레 스터디](https://www.dalestudy.com/) 웹사이트를 만들었습니다. 덕분에 2기 때는 웹사이트를 통해서 스터디 참여 방법을 안내하고 멤버를 모집할 수가 있었죠. 1기 때 협업 내용은 [GitHub 프로젝트](https://github.com/orgs/DaleStudy/projects/2)에 상세하게 기록되어 있으며, Tech Lead를 맡으셨던 @sounmind 님의 [LinkedIn 포스팅](https://www.linkedin.com/posts/suhyeong-evan-lee_im-excited-to-share-a-unique-side-project-ugcPost-7226003250871775232-Amz1/?utm_source=share&utm_medium=member_desktop)을 통해서도 엿볼 수 있습니다.

2기에서는 어떤 프로젝트를 진행할지는 아직 결정하지 않았으나 1기 사이드 프로젝트를 회고할 때 나왔던 [아이디어 목록](https://github.com/DaleStudy/leetcode-website/issues/193)에서 하나를 선정하거나 더 좋은 아이디어를 건의해주시면 함께 고려해볼 수 있을 것 같습니다. 그 밖에 프로젝트 기간과 진행 방식, 역할 분담, 기술 스택과 같은 구체적인 부분은 아직 정해진 바가 없으며 팀원이 확정되는 대로 Kick-off 미팅 때 하나씩 같이 얘기해보면 좋을 것 같습니다.

사이드 프로젝트이지만 그래도 배워가시는 게 있도록 17년차 개발자인 제가 신경써서 마치 실제로 원격 근무하는 회사처럼 프로젝트 관리를 해드릴 예정입니다. 스터디 1기 때처럼 성공적인 프로젝트를 위해서 모든 분들께 기회를 드리기는 어려울 것 같아요. 최소 신청 자격을 **답안 PR을 4회 이상 머지**, **정기 모임에 2회 이상 참석**으로 제한하오니 양해 부탁드립니다.

프로젝트 참가를 희망하시는 분들은 아래 댓글로 신청해주시면 감사하겠습니다.

**Comment by DaleSeo** [comment]
@DaleStudy/website 현재 달레 스터디 웹사이트를 유지보수해주시고 계신 1기 사이드 프로젝트 멤버 분들은 무조건 환영입니다용~  😉

**Comment by TonyKim9401** [comment]
Tony 신청합니다.

GitHub: TonyKim9401
Discord: TonyKim9401

**Comment by wogha95** [comment]
프로젝트 참가 희망합니다~!!

Github: wogha95
Discord: 재호

**Comment by HC-kang** [comment]
어떤걸 하게 될지는 모르겠지만, 재미있을 것 같네요!

Github: HC-kang
Discord: HC-kang

**Comment by SamTheKorean** [comment]
프로젝트 참가 희망합니다! 

Github: samthekorean
Discord: Sam

**Comment by yolophg** [comment]
저도 1기에 이어 참가 희망합니다!
Github : yolophg
Discord : Helena

**Comment by obzva** [comment]
참가 희망합니다
Github: obzva
Discord: Flynn

**Comment by sounmind** [comment]
와~ 참가 희망합니다. 앞으로 또 어떤 경험을 할지 두근거리네요.
Github: sounmind
Discord: Evan

**Comment by Sunjae95** [comment]
참가 희망합니다
Github: Sunjae95
Discord: 선재

**Comment by DaleSeo** [comment]
충분한 팀원이 모아져서 신청을 마감하겠습니다!
